test/test_pointevaluation.jl

```diff
@@ -1,4 +1,4 @@
-using Ferrite, Test
+using Ferrite, Test, Logging, OrderedCollections
```

Comment:
Why is this even here, I don't think we do this consistently?

Reply:
I am not sure why these are missing from the test. Running only parts of the tests is right now quite painful, as we include some of the deps in the runner and sometime some of the deps in the file containing the testsets. Would you agree that it would be better if the different files containing the test sections would be self-contained, so we can run them directly for testing instead of modifying the test codes every time we only want to test part of the API?

Reply:
I totally agree that this can be very useful, I was just not aware that we have this as a convention to do! 

Would of course then be nicer if it would error if one doesn't do that, otherwise it is quite hard to make sure it is done consistently. This also includes e.g. including `test_utils.jl`. Perhaps it could rather make sense to have a file `initialize_tests.jl` that is included at the beginning of `runtests.jl`, and running individual tests can then always be done by including that file before the specific file (or snippets of code)? Of course a separate discussion...

Reply:
Yea we should probably follow something like this here https://github.com/SciML/OrdinaryDiffEq.jl/blob/1804a1bbc436c88580310316e69ff5a091f65eb1/test/runtests.jl , especially if we start expanding convergence rate tests and GPU test coverage.